Calendars/Calendars.Plugin.iOSUnified/CalendarsImplementation.cs

```diff
@@ -154,7 +154,9 @@ public async Task AddOrUpdateCalendarAsync(Calendar calendar)
                 calendar.ExternalID = deviceCalendar.CalendarIdentifier;
 
                 // Update color in case iOS assigned one
+#if !NET6_0
```

Comment:
nit: You can use just `NET` here. It's only defined in `net5` and `net6` 
```suggestion
#if !NET
```